cli_tools/daisy_common/daisy_utils.go

```diff
@@ -0,0 +1,82 @@
+//  Copyright 2018 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package daisycommon
+
+import (
+	"cloud.google.com/go/compute/metadata"
+	"context"
+	"fmt"
+	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"
```

Comment:
Separate the standard library imports from the external imports with a blank line.

Reply:
Is cloud import considered external import in this case? E.g. 

	"cloud.google.com/go/compute/metadata"

	"context"
	"fmt"

	"github.com/GoogleCloudPlatform/compute-image-tools/daisy"

Reply:
"context"
"fmt"

"cloud.google.com/go/compute/metadata"
"github.com/GoogleCloudPlatform/compute-image-tools/daisy"